src/middlewares/isAdmin.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
```

Comment:
I think you branched this branch "feature/isAdminMiddleware" from "feature/isLoggedMiddleware" not from "main" maybe you would have to empty the file "isLogged.js" in this branch to merge with no problems

Reply:
or you can merge "feature/isLoggedMiddleware" into "feature/isAdminMiddleware" then delete "feature/isLoggedMiddleware" 
lastly merge "feature/isAdminMiddleware" to "main"